packages/block-library/src/navigation-menu-item/edit.js

```diff
@@ -84,8 +85,11 @@ function NavigationMenuItemEdit( {
 			</div>
 		);
 	} else {
-		content = attributes.label;
+		content = <div className="wp-block-navigation-menu-item__container">
```

Comment:
We can't AFAIK.
I've added this CSS class in order to tackle [this issue](https://github.com/WordPress/gutenberg/pull/17832#issuecomment-542651224). Let me try to expling it. The visual issue appears when the user add/edit sub items, generating something like the following markup/css:


```html
<div class="wp-block-navigation-menu-item is-editing">
    <div class="wp-block-navigation-menu-item__container">
        Item Label
    </div>
    <div class="inner-blocks">
        <div>
            <div>
                <div class="wp-block-navigation-menu-item is-editing">
                    <div class="wp-block-navigation-menu-item__container">
                        <textarea class="block-editor-plain-text">
                            Editing sub -item
                        </textarea>
                </div>
            </div>
        </div>
    </div>
</div>
```


```scss
.wp-block-navigation-menu-item {
	font-family: $editor-font;
	font-weight: bold;
	font-size: $text-editor-font-size;

	.wp-block-navigation-menu-item__container,
	&.is-editing .editor-plain-text {
		background-color: var(--background-color-menu-link);
		color: var(--color-menu-link);
	}
}
```

The element with `.wp-block-navigation-menu-item` wraps all elements which comform the item, including those ones needed to set up the menu. If we apply the background color straigth to this class, it will paint it to the whole surface. In another hand, `.wp-block-navigation-menu-item__container`, wraps the `<label />` or `<textarea />` in editing mode, allowing apply the colors rightly.


The follwwing image shows in `green` the `.wp-block-navigation-menu-item` elements, and in `red` the `.wp-block-navigation-menu-item__container` ones.

<img width="924" alt="Screen Shot 2019-10-17 at 8 47 48 AM" src="https://user-images.githubusercontent.com/77539/67006365-5bba5700-f0bb-11e9-8154-4a69b6d8c410.png">

Just for testing purpose, I've applied background color blue straigh to the `.wp-block-navigation-menu-item` to show how it behaves:

<img width="1577" alt="Screen Shot 2019-10-17 at 9 01 26 AM" src="https://user-images.githubusercontent.com/77539/67007062-f5363880-f0bc-11e9-8f2b-bca5bb2c1885.png">